libkbfs/tlf_journal.go

```diff
@@ -931,7 +933,10 @@ func (j *tlfJournal) clearMDs(ctx context.Context, bid BranchID) error {
 }
 
 func (j *tlfJournal) wait(ctx context.Context) error {
-	// TODO: figure out if the journal is currently paused; if so,
-	// return an error.
-	return j.wg.Wait(ctx)
+	workLeft, err := j.wg.WaitUnlessPaused(ctx)
+	if err == nil && workLeft {
```

Comment:
i had to think about this for a while. I think it's clearer with the usual pattern:

```
workLeft, err := ...
if err != nil {
  return err
}
if workLeft {
  j.log...
}
return nil
```